sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -221,9 +221,10 @@ void gnrc_sixlowpan_nd_rtr_sol_reschedule(gnrc_ipv6_nc_t *nce, uint32_t sec_dela
 {
     assert(nce != NULL);
     assert(sec_delay != 0U);
+    gnrc_ipv6_netif_t *iface = gnrc_ipv6_netif_get(nce->iface);
     vtimer_remove(&nce->rtr_sol_timer);
     vtimer_set_msg(&nce->rtr_sol_timer, timex_set(sec_delay, 0), gnrc_ipv6_pid,
-                   GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL, nce);
+                   GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL, iface);
```

Comment:
Ah sorry... You are right… I thought it was changing `GNRC_SIXLOWPAN_ND_MSG_MC_RTR_SOL`. But then the timer needs to be changed to `iface->rtr_sol_timer`, too.